core/src/test/java/it/pagopa/selfcare/mscore/core/InstitutionServiceImplTest.java

```diff
@@ -747,6 +747,16 @@ void testUpdateInstitution7() {
         verify(userService).checkIfInstitutionUser(any(), any());
     }
 
+    /**
+     * Method under test: {@link InstitutionServiceImpl#updateInstitutionDelegation(String, boolean)} (String, InstitutionUpdate, String)}
+     */
+    @Test
+    void testUpdateInstitutionDelegation() {
+        assertDoesNotThrow(
+                () -> institutionServiceImpl.updateInstitutionDelegation("42", true));
+        verify(institutionConnector).findAndUpdate(any(), any(), any(), any());
```

Comment:
you could verify using two ArgumentCaptor that `findAndUpdate` is called using institutionId and an InstitutionUpdate with delegation true....